kimchi/src/prover_index.rs

```diff
@@ -26,25 +26,25 @@ pub struct ProverIndex<G: CommitmentCurve> {
 
     /// The symbolic linearization of our circuit, which can compile to concrete types once certain values are learned in the protocol.
     #[serde(skip)]
-    pub linearization: Linearization<Vec<PolishToken<ScalarField<G>>>>,
+    pub(crate) linearization: Linearization<Vec<PolishToken<ScalarField<G>>>>,
 
     /// The mapping between powers of alpha and constraints
     #[serde(skip)]
-    pub powers_of_alpha: Alphas<ScalarField<G>>,
+    pub(crate) powers_of_alpha: Alphas<ScalarField<G>>,
 
     /// polynomial commitment keys
     #[serde(skip)]
     pub srs: Arc<SRS<G>>,
 
     /// maximal size of polynomial section
-    pub max_poly_size: usize,
+    pub(crate) max_poly_size: usize,
 
     /// maximal size of the quotient polynomial according to the supported constraints
-    pub max_quot_size: usize,
+    pub(crate) max_quot_size: usize,
```

Comment:
This field is only being used by `verifier_index` to extract its own `max_quot_size`, which is being used nowhere (as pointed out later). Shall we remove them?